src/cv/Cv.js

```diff
@@ -48,100 +50,58 @@ class Cv extends Component {
-        headerLogo = frontEnd;
+        sourcesFirst.headerLogo = frontEnd;
       } else if (title.includes('java')) {
-        headerLogo = java;
+        sourcesFirst.headerLogo = java;
       } else if (title.includes('test')) {
-        headerLogo = test;
+        sourcesFirst.headerLogo = test;
       } else if (title.includes('bussiness')) {
-        headerLogo = bussinessAnalyst;
+        sourcesFirst.headerLogo = bussinessAnalyst;
       } else if (title.includes('cq5')) {
-        headerLogo = cq5;
+        sourcesFirst.headerLogo = cq5;
       }
     }
 
 
     return (
       <div className="Cv">
-        <div className="Cv-page" id="divToPrintPage1">
-          <div className="Cv-title">
-            <img src={logoDS} className="Cv-logo" alt="logo" />
-            {headerLogo ? <img src={headerLogo} className="Cv-logo-header" alt="logo" /> : ''}
-            <div className="text-main">{user.name}</div>
-            <div className="text-second">{user.title}</div>
-          </div>
-          <div className="Cv-experience">
-            <div className="Cv-experience-title">
-              <img src={experience} className="Cv-logo" alt="logo" />
-              <span>EXPERIENCE</span>
-            </div>
-            {experiencesList}
-          </div>
-          <div className="Cv-education">
-            <div className="Cv-education-title">
-              <img src={education} className="Cv-logo" alt="logo" />
-              <span>EDUCATION</span>
-            </div>
-            {educationList}
-          </div>
-        </div>
-        <div className="Cv-page" id="divToPrintPage2">
-          <div className="Cv-expectations">
-            <div className="Cv-expectations-title">
-              <img src={target} className="Cv-logo" alt="logo" />
-              <span>PROFESIONAL EXPECTATION</span>
-            </div>
-            <div className="Cv-expectations-desc">
-              <span>{user.professionalExpectations}</span>
-            </div>
-          </div>
-          <div className="Cv-skillset">
-            <div className="Cv-skillset-logo">
-              <img src={skillset} className="Cv-logo" alt="logo" />
-            </div>
-            <div className="Cv-skillset-title">
-              <span>SKILLSET</span>
-            </div>
-            {skillsetList}
-          </div>
-          <div className="Cv-personalNote">
-            <div className="Cv-personalNote-title">
-              <img src={profile} className="Cv-logo" alt="logo" />
-              <span>PERSONAL NOTE</span>
-            </div>
-            <div className="Cv-personalNote-desc">
-              {user.personalNote}
-            </div>
-          </div>
-          <div className="Cv-contact">
-            <img src={bluePartCircle} className="Cv-logo" alt="logo" />
-            <span>www.ds.eu</span>
-          </div>
-        </div>
+        <PageFirst user={user} educationList={educationList} experiencesList={experiencesList} sources={sourcesFirst} />
```

Comment:
Same here, we can move every list to pages and pass only user or part of user data.